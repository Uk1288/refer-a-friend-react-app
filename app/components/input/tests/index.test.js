import React from 'react';
import { render } from 'react-testing-library';
import Input from '../index';

describe('<Input />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Input />);
    expect(firstChild).toMatchSnapshot();
  });
});
