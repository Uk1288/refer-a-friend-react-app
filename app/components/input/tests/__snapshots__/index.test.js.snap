// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render and match the snapshot 1`] = `
<input
  class="input__StyledInput-sc-10zxn30-0 lkoULL"
/>
`;
